go.mod

```diff
@@ -5,4 +5,5 @@ go 1.14
 require (
 	k8s.io/apimachinery v0.18.8
 	k8s.io/client-go v0.18.8
+	k8s.io/klog v1.0.0
```

Comment:
(nit, feel free to ignore) looks like this is just added by `go mod tidy` and not really related, could be broken into a separate commit

Reply:
Fixed.